README.md

```diff
@@ -6,15 +6,15 @@
 [![Documentation](https://docs.rs/pathfinding/badge.svg)](https://docs.rs/pathfinding)
 [![License: Apache-2.0/MIT](https://img.shields.io/crates/l/pathfinding.svg)](#license)
 
-This crate implements several pathfinding and flow algorithms in [Rust]():
+This crate implements several pathfinding and flow algorithms in [Rust][Rust]
 
-- [A*](): find the shortest path in a weighted graph using an heuristic to guide the process.
-- [breadth-first search (BFS)](): explore nearest neighbours first, then widen the search.
-- [depth-first search (DFS)](): explore a graph by going as far as possible, then backtrack.
-- [Dijkstra](): find the shortest path in a weighted graph.
-- [Edmonds Karp](): find the maximum flow in a directed graph.
-- [Fringe](): find the shortest path in a weighted graph using an heuristic to guide the process.
-- [IDA*](): explore longer and longer paths in a weighted graph at the cost of multiple similar examinations.
+- [A*][A*]: find the shortest path in a weighted graph using an heuristic to guide the process.
+- [breadth-first search (BFS)][BFS]: explore nearest neighbours first, then widen the search.
+- [depth-first search (DFS)][DFS]: explore a graph by going as far as possible, then backtrack.
+- [Dijkstra][Dijkstra]: find the shortest path in a weighted graph.
+- [Edmonds Karp][Edmonds Karp]: find the maximum flow in a directed graph.
+- [Fringe][Fringe]: find the shortest path in a weighted graph using an heuristic to guide the process.
+- [IDA*][IDA*]: explore longer and longer paths in a weighted graph at the cost of multiple similar examinations.
 
```

Comment:
I agree that BFS and DFS are wrong and require using `[BFS]` and `[DFS]` instead of `()`. However, I am surprised that `[Dijkstra]` (standalone, without `()`) doesn't work, so thanks for those fixes.